src/systems/components-queue.js

```diff
@@ -0,0 +1,51 @@
+AFRAME.registerSystem("components-queue", {
```

Comment:
Don't love this name but I don't have better suggestions heh. `throttled-tick`, `tick-queue`, `behavior-queue`?